website/blog/2022-02-22-towards-better-releases.md

```diff
@@ -0,0 +1,57 @@
+---
+author: Vadim Chelyshov
+title: Towards better releases
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+As many of you might have noticed, the previous `0.11.0` release didn't go smoothly due to a number of issues that came to light only after the release was published.
+Some of them were quite critical and the only option to continue to work was to downgrade Metals until `0.11.1` arrived.
+
+We apologize for that!
+
+However, this post is not only about saying sorry.
+The reasons why it's happened weren't a complete surprise for us.
+In order to avoid such situations in the future we have taken upon us to take steps that will allow us to detect issues early.
```

Comment:
```suggestion
In order to avoid such situations in the future we have taken it upon ourselves to take steps that will allow us to detect issues earlier.
```